lms/templates/learner_dashboard/course_card_actions.underscore

```diff
@@ -0,0 +1,14 @@
+<% if(enrollment_status){ %>
+    <div class="enrollment-info"><%- gettext('ENROLLED') %></div>
+    <a id="view_course_link" href="<%- course_url %>" class="btn-neutral btn-large view-course-link">
+        <%- gettext('View Course') %>
+    </a>
+<% }else{ %>
+    <div class="enrollment-info"><%- gettext('NOT ENROLLED') %></div>
+    <div class="js-user-cta run-mode-select"></div>
+    <div class="enroll-button-container">
+        <button id="enroll_button" type="button" class="btn-brand btn-large enroll-button">
```

Comment:
We should not use ids unless absolutely necessary because of the constraint that they be unique.